### DISCUSSION TITLE
ULTRALYTICS HUB x ROBOFLOW

### DISCUSSION THREAD

**Discussion by taliabender** [ultralytics/yolov5#10425] [Show and tell]
**Ultralytics Live Sessions, Ep. 2: Ultralytics HUB x Roboflow 🔥**

Join [Joseph Nelson](https://www.linkedin.com/in/ACoAAA8Pi9wBn5BP70RvfxZvfrsO-BTUG888c4E) from [Roboflow](https://www.linkedin.com/company/roboflow-ai/), and [Glenn Jocher](https://www.linkedin.com/in/ACoAAASa684BOUpWeSgSJbXKXU7MaWJqS4RCFes) on Tuesday, December 13th at 19:00 CET, to learn how to upload your data faster and more seamlessly using Ultralytics HUB!

Secure your spot here! https://bit.ly/3H9vujw

![HUB x   ##Roboflow](https://user-images.githubusercontent.com/85292283/205996456-bf3efa33-9c46-455e-b322-a64886cc7a0b.png)

**Comment by glenn-jocher** [comment]
@taliabender awesome!! This is going to be fun :)
